be/src/olap/olap_define.h

```diff
@@ -57,6 +57,9 @@ static const uint16_t OLAP_VARCHAR_MAX_LENGTH = 65535;
 // the max length supported for string type 2GB
 static const uint32_t OLAP_STRING_MAX_LENGTH = 2147483647;
 
+// the max length supported for json type 2GB
+static const uint32_t OLAP_JSON_MAX_LENGTH = 10 * 1024 * 1024;
```

Comment:
It seems `10 * 1024 * 1024` do not equal `2GB`

Reply:
nice catch, change 2G to 10M